### ISSUE TITLE
[Bug] iOS app crashes after some time (EXC_BAD_ACCESS)

### ISSUE THREAD

**Issue by D33pTh0ught** [Unity-Technologies/arfoundation-samples#716]
**Description**
After running our AR scene for some time (on iOS) the app crashes with a segfault (EXC_BAD_ACCESS).
The issue appears to surface faster if mobileMTRendering is enabled for iOS.
The android app does not crash.

**To Reproduce**
1. Clone https://github.com/artfabrik/arfoundation-samples.git
2. Build and run (debug) using XCode 12.3 (12C33) and iPhone OS 14.2 (Build 18B92) (Model: iPhone XR)
3. After some time (can range from 2min to 2h) the app crashes.

**Expected behavior**
The app should not crash.

**Actual behavior**
The app crashes after some time even without interacting with the device (even when the scene only relies on scripts from the arfoundation-samples).

Here are some crash logs of executions with and without mobileMTRendering enabled for iOS:
[logs.zip](https://github.com/Unity-Technologies/arfoundation-samples/files/5770675/logs.zip)

**Smartphone:**
 - Device: iPhone XR (reported as iPhone11,8 in XCode Devices and Simulators log)
 - OS: iPhone OS 14.2 (Build 18B92)
 - Unity version 2020.1.17.f1 (also seems to happen on older versions)
 - ARFoundation version 4.1.1 (also seems to happen on older versions)

**Comment by tdmowrer** [comment]
This appears to be a problem introduced in iOS 14. We expect it to be fixed in a future iOS release.

**Comment by D33pTh0ught** [comment]
Thanks for the response, do you know if they are aware of the issue?

**Comment by D33pTh0ught** [comment]
This issue should not be closed as it is not fixed yet.

**Comment by tdmowrer** [comment]
There is nothing more that we can do. This is an issue in iOS.

**Comment by D33pTh0ught** [comment]
I am aware of that. Would you know of any other way to track the progress on this issue and/or is there still the need to create a bug report on their end? If so what would be the best way to do so?

**Comment by tdmowrer** [comment]
If you log an issue at https://feedbackassistant.apple.com/, Apple can notify you when it is fixed.

**Comment by D33pTh0ught** [comment]
Thanks, i logged an issue there.

**Comment by phields** [comment]
> Thanks, I logged an issue there.

I got the same issue, any suggestion?

**Comment by D33pTh0ught** [comment]
> I got the same issue, any suggestion?

We noticed that after updating ARFoundation and Unity (e.g. Unity 2020.1 with ARFoundation 4.1) the crash does not occur that fast and that often. However it is very difficult and tedious to evaluate under what circumstances exactly this issue is triggered since it sometimes only surfaces after an hour or more.
Unfortunately I did not get any response from apple.
